### PR TITLE
Added training/cora.py to load the Cora dataset, updated training/cit…

### LINKED ISSUE #5: Data Processing : Import and Preprocess Cora
Write a python script to import and preprocess the Cora dataset using:

- PyTorch Geometric

Pre-process using:
- Feature normalization (communicate on normalization parameters)
- Generate dataset splits (training / test / validation)
- Use PyG data format

### PR DESCRIPTION
**Pull request by BrandonN24**
Resolves #5 

Features:
- training/cora.py Python script to load the Cora dataset for use in the training scripts to be implemented.
- Added a unit test for training/cora.py
- Adjusted the root directory of the dataset in citeseer.py

Limitation:
- This commit does not split the Cora dataset into the training / test / validation splits. This will be done in issue #9 and issue #10.

### CODE REVIEW

**Review by simonbruce85 — APPROVED** [review]
lgtm
